src/Util/Frame.php

```diff
@@ -49,7 +60,7 @@ public function getFile()
     }
 
     /**
-     * @return int|null
+     * @return integer|null
```

Comment:
> The upcoming PSR (and phpdocumentor) allows both, I think using full type name is more straightforward.

That standard also points out that "int" is correct, and "integer" is not, but is allowed for the moment.